source/hassio/index.markdown

```diff
@@ -1,19 +1,19 @@
 ---
-title: "Hass.io"
+title: "Home Assistant"
 description: "Manage your Home Assistant and custom add-ons."
 redirect_from: /integrations/hassio/
 ---
 
-Hass.io turns your Raspberry Pi (or another device) into the ultimate home automation hub powered by Home Assistant. With Hass.io you can focus on integrating your devices and writing automations.
+Home Assistant turns your Raspberry Pi (or another device) into the ultimate home automation hub. With Home Assistant, you can focus on integrating your devices and writing automations.
 
 [Go to the installation instructions &raquo;][install]
 
-The advantages of using Hass.io:
+The advantages of using Home Assistant:
 
 - Free and open source
 - Optimized for embedded devices like Raspberry Pi
 - 100% local home automation
-- Easy installation and updates (powered by [HassOS] and [Docker])
+- Easy installation and updates (powered by [Home Assistant] and [Home Assistant Core])
```

Comment:
HassOS is now the Home Assistant Operating System. Furthermore, those are links, which are now broken.
You need to update the references.

Reply:
Im not sure how to properly fix links. After adding my adjustments of [Home Assistant], do i just change the reference in the footer to this: [Home Assistant]: https://github.com/home-assistant/homeassistant  ?

Reply:
Yes, that is about it.

For reference, you could look upon how you could create Markdown links (there are multiple methods). (Tons of Markdown guides to be found on Google 😉 )